src/MedPrep.Api/Controllers/VideoUploadController.cs

```diff
@@ -40,4 +40,95 @@ public async Task<ActionResult<VideoUploadResponse>> UploadVideo([FromForm] Vide
 
         return this.Ok(response);
     }
+
+    [HttpGet]
+    [Route("id")]
```

Comment:
You in order to specify the id template in the route you can write this:
```csharp
[Route("{id}")]
```
To make it more concise you can just do this and get rid of the Route attribute entirely
```csharp
[HttpGet("{id}")]
```